CHANGELOG.md

```diff
@@ -75,7 +75,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed an issue where author list parser can't generate bibtex for Chinese author. [#4169](https://github.com/JabRef/jabref/issues/4169)
 - We fixed an issue where the list of XMP Exclusion fields in the preferences was not be saved [#4072](https://github.com/JabRef/jabref/issues/4072)
 - We fixed an issue where the ArXiv Fetcher did not support HTTP URLs [#4367](https://github.com/JabRef/jabref/pull/4367)
-
+- We fixed an issue where multiple radiobuttons in preference menu belonging to the same group were selectable simultaneously [#4409](https://github.com/JabRef/jabref/issues/4409)
```

Comment:
Please remove the changelog entry. This bug was introduced in one of the recent development versions and was not present in one of the previous releases.